--- /home/taemin/.fzf/shell/key-bindings.bash	2017-11-13 09:30:22.228071504 +0900
+++ settings/fzf/key-bindings.bash	2017-11-13 09:29:45.450958944 +0900
@@ -72,11 +72,11 @@
 
   # CTRL-T - Paste the selected file path into the command line
   if [ $BASH_VERSINFO -gt 3 ]; then
-    bind -x '"\C-t": "fzf-file-widget"'
+    bind -x '"\ei": "fzf-file-widget"'
   elif __fzf_use_tmux__; then
-    bind '"\C-t": " \C-u \C-a\C-k`__fzf_select_tmux__`\e\C-e\C-y\C-a\C-d\C-y\ey\C-h"'
+    bind '"\ei": " \C-u \C-a\C-k`__fzf_select_tmux__`\e\C-e\C-y\C-a\C-d\C-y\ey\C-h"'
   else
-    bind '"\C-t": " \C-u \C-a\C-k`__fzf_select__`\e\C-e\C-y\C-a\C-y\ey\C-h\C-e\er \C-h"'
+    bind '"\ei": " \C-u \C-a\C-k`__fzf_select__`\e\C-e\C-y\C-a\C-y\ey\C-h\C-e\er \C-h"'
   fi
 
   # CTRL-R - Paste the selected command from history into the command line
